ci/integration_tests/src/tests/cw20_stake_test.rs

```diff
@@ -0,0 +1,120 @@
+use core::time;
+use std::thread;
+
+use crate::helpers::{chain::Chain, helper::create_dao};
+use cosmwasm_std::{to_binary, Uint128};
+use cw20_stake::{msg::StakedValueResponse, state::Config};
+use cw_core_interface::voting::VotingPowerAtHeightResponse;
+use test_context::test_context;
+
+// #### ExecuteMsg #####
+
+#[test_context(Chain)]
+#[test]
+fn execute_stake_tokens(chain: &mut Chain) {
+    let voting_contract = "cw20_staked_balance_voting";
+
+    let res = create_dao(chain, None, "exc_stake_create_dao", chain.user.addr.clone());
+    assert!(res.is_ok());
```

Comment:
Suggest that we remove assertions like this. With them we don't get to see the error that occured in tests. Better to just unwrap so the panic has that information inside it.

Reply:
Great point. I also updated the Display impl for this CosmosSDK error to actually include the relevant base error.